.changes/v3.10.0/1076-features.md

```diff
@@ -0,0 +1,4 @@
+* **New Resource:** `vcd_nsxt_distributed_firewall_rule` to manage NSX-T Distributed Firewall one by
```

Comment:
I feel like we should state that this is above API functionality here too.

Reply:
Added extra line

Reply:
Added to resource information as well.
<img width="723" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/a22608c9-eb51-4383-801f-ba4413bbf8de">

Reply:
Very useful! Here's a polishing suggestion. Changes highlighted or code-quoted:

> "Multiple rules defined with this resource will not be created in parallel, because **CD** API provides no direct endpoint to create a single rule. **To overcome this,** `vcd_nsxt_distributed_firewall_rule` **calls an "update all rules" API endpoint for each single rule, and this call is serialized to avoid data conflicts.**

Reply:
I have patched and read this again. The only thing I changed in your quote is to expand CD to Cloud Director. It sounded a bit cryptic as we don't use this abbreviation too much in our docs.